Source/WebKit/NetworkProcess/Downloads/cocoa/DownloadCocoa.mm

```diff
@@ -148,6 +148,16 @@
     if (enableModernDownloadProgress()) {
         bool isUsingPlaceholder = useDownloadPlaceholder == WebKit::UseDownloadPlaceholder::Yes;
 
+        if (!m_downloadTask.get().currentRequest.URL) {
```

Comment:
Why are we checking m_downloadTask.get().currentRequest.URL, it doesn't seem to be used below.

Reply:
I included this check here, since this URL is referenced inside `-[WKModernDownloadProgress initWithDownloadTask:]`.

Thanks for reviewing!